src/styles/blocks/box.scss

```diff
@@ -0,0 +1,7 @@
+.box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
```

Comment:
You dont need to set width:100% as by default blocks will take 100% of space they can